### PR TITLE
Lookup: Placeholder shouldn't be removed when the items option is changed (T1099804)

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Comment by ksercs** [comment]
Reading this pull request name I cannot understand: it is a problem that placeholder is removed? Or it should be removed and it is an expected behavior?

File: testing/testcafe/tests/editors/autocomplete/common.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import createWidget from '../../../helpers/createWidget';
+import { restoreBrowserSize } from '../../../helpers/restoreBrowserSize';
+
+fixture`Autocomplete_placeholder`
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async (t) => {
+    await restoreBrowserSize(t);
+  });
+
+test('The placeholder is visible after changing of items option when value is not choosen', async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/autocomplete/common.ts]:
```suggestion
test('Placeholder is visible after items option change when value is not chosen (T1099804)', async (t) => {
```

File: testing/testcafe/tests/editors/autocomplete/common.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import Autocomplete from '../../../model/autocomplete';
+import createWidget from '../../../helpers/createWidget';
+import { restoreBrowserSize } from '../../../helpers/restoreBrowserSize';
+
+fixture`Autocomplete_placeholder`
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async (t) => {
+    await restoreBrowserSize(t);
```
Comment (ksercs) [inline testing/testcafe/tests/editors/autocomplete/common.ts]: Do we really need a functional test for this scenario? Seems like unit test can be written easy as well

**Review by ksercs — APPROVED** [review]

File: testing/testcafe/tests/editors/autocomplete/common.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/autocomplete/common.ts]: Yes, there are no testcafe tests for placeholder.

**Review by Zedwag — APPROVED** [review]

**Pull request merged**